app/models/event_schedule.rb

```diff
@@ -0,0 +1,7 @@
+class EventSchedule < ApplicationRecord
+  validates_presence_of :event_name
```

Comment:
This will help us make sure the event can be scheduled on the calendar. Great job.